internal/common/backoff/retrypolicy.go

```diff
@@ -28,25 +28,19 @@ import (
 	"math"
 	"math/rand"
 	"time"
+
+	"go.temporal.io/sdk/internal/common/retry"
 )
 
 const (
-	// NoInterval represents Maximim interval
-	NoInterval                      = 0
```

Comment:
Technically, NoInterval and DefaultBackoffCoefficient are accessible by the user. Can this be a breaking change?

Reply:
We don't provide any guarantees about backwards compatibility within internal package, so this should be fine.